webapp-django/crashstats/signature/jinja2/signature/signature_summary.html

```diff
@@ -1,3 +1,58 @@
+{% from "macros/signature_startup_icons.html" import startup_crash_icon,
+                                                     potential_startup_crash_icon,
+                                                     potential_startup_window_crash_icon,
+                                                     hang_crash_icon, plugin_crash_icon,
+                                                     browser_crash_icon %}
+
+{% if signature_stats %}
+<div class="panel tab-inner-panel crash-type-indicators-panel
+            {% if signature_stats.is_startup_related_crash %} startup-crash-warning {% endif %}">
+    {% if signature_stats.is_startup_crash %}
+    <div class="crash-type-indicator-container">
+        {{ startup_crash_icon() }}
+        Startup Crash, all crashes happened during startup
+    </div>
+    {% endif %}
+
+    {% if signature_stats.is_potential_startup_crash %}
+    <div class="crash-type-indicator-container">
+        {{ potential_startup_crash_icon(
+            num_startup_crashes=signature_stats.num_startup_crashes,
+            num_crashes=signature_stats.num_crashes,
+        ) }}
+        Potential Startup Crash, {{ signature_stats.num_startup_crashes }}
+        out of {{ signature_stats.num_crashes }} crashes happened during startup
+    </div>
+    {% endif %}
```

Comment:
I think we should show either `is_startup_crash` OR `is_potential_startup_crash` OR neither of them, but never both. I'd change this block to be an `else if` instead of an `if`.

Reply:
The way the two properties are set up right now, they should be exclusive and never be true at the same time. `is_potential_startup_crash` needs to be under while `is_startup_crash` needs to be equal. I guess the wording makes it confusing, since one would expect a "potential startup crash" to also be a "startup crash".

With that in mind, would you make changes? I could rename `is_startup_crash` to `is_always_startup_crash`.

Reply:
Yeah, that is what I would currently expect. The behavior should be the same as the current `topcrashers` page. The second icon red rocket icon is for a crash within one minute of launch, as opposed to pure startup.
So there are three startup crash icons:

1. All crashes happened directly during start up.
2. Some crashes happened directly during start up (should be exclusive with 1).
3. Some crashes happened during one minute of launch. This looks at a different source statistic than 1 and 2.

The user messaging for `#3` currently says `Potential Startup Crash`. I am using `is_startup_window_crash` in the code. I would personally change the user messaging to be `Potential Startup Window Crash` for better clarity. Do you think that is a change I could make?

Reply:
Ahh... ok. I see what's going on now. I think this is fine. Thank you for clarifying!